sqlglot/dialects/duckdb.py

```diff
@@ -369,6 +380,11 @@ class Parser(parser.Parser):
 
         FUNCTIONS_WITH_ALIASED_ARGS = {*parser.Parser.FUNCTIONS_WITH_ALIASED_ARGS, "STRUCT_PACK"}
 
+        SHOW_PARSERS = {
+            "TABLES": _show_parser("TABLES"),
+            "ALL TABLES": _show_parser("TABLES"),
```

Comment:
This isn't correct, `SHOW TABLES` ≠ `SHOW ALL TABLES`:
```suggestion
            "ALL TABLES": _show_parser("ALL TABLES"),
```